packages/katello/katello/katello-backup

```diff
@@ -1,18 +1,10 @@
-#!/usr/bin/env ruby
+#!/bin/sh
 
-# check if production install
-if __FILE__.include?('sbin')
-  UTILS_PATH='/usr/share/katello'
-else
-  UTILS_PATH=File.expand_path('..', __FILE__)
-end
+MESSAGE="
+IMPORTANT: katello-backup has been removed in favor of 'foreman-maintain backup'.
+This change happened in order to support remote databases and a newer mongo version.
+Please update your scripts and/or cron jobs.
```

Comment:
How different is the syntax for foreman-maintain backup? I am wondering if for a release we can change this script to essentially do:

```
foreman-maintain backup $@
```

To pass through the options and keep the script "intact" for a single release.

Reply:
we discussed that, it would require a "translation" script since the syntax is different for foreman-maintain backup. It uses subcommands for each backup. i.e.

`foreman-maintain backup online`
is 
`katello-backup --online-backup`

I'm not sure if its worth the added time and complexity to do something like that. I plan on adding a deprecation warning in the next 3.6.z if that helps at all.